src/actions/index.js

```diff
@@ -22,3 +23,9 @@ export const hoverOutAction = (cardId) => {
     id: cardId
   };
 };
+
+export const checkedJointAccount = () => {
+  return {
+    type: CHECKED
```

Comment:
Probably not a great actionType. Would you use this action for anything other than checking the joint account? Any reason not to use redux-form for this?